Code/GraphMol/Depictor/EmbeddedFrag.cpp

```diff
@@ -419,6 +419,43 @@ bool EmbeddedFrag::matchToTemplate(const RDKit::INT_VECT &ringSystemAtoms,
   return true;
 }
 
+// find any atoms in the ring that are in trans double bonds
+static RDKit::INT_VECT findTransRingAtoms(const RDKit::ROMol& mol, const RDKit::INT_VECT& ring)
+{
+  RDKit::INT_VECT res;
+  for (size_t i=0; i<ring.size(); ++i) {
+    const auto atom1 = ring[i];
+    const auto atom2 = ring[(i + 1) % ring.size()];
+    const auto b = mol.getBondBetweenAtoms(atom1, atom2);
+    if (b->getBondType() != RDKit::Bond::DOUBLE) {
+      continue;
+    }
+    const auto stype = b->getStereo();
+    if (stype <= RDKit::Bond::STEREOANY) {
+      continue;
+    }
+
+    // We care about bonds that are trans with respect to this ring
+    const auto& neighbors = b->getStereoAtoms();
+    if (neighbors.size() != 2) {
+      continue;
+    }
+    const auto leftIsIn = std::find(ring.begin(), ring.end(), neighbors[0]) != ring.end();
+    const auto rightIsIn = std::find(ring.begin(), ring.end(), neighbors[1]) != ring.end();
+    if (stype == RDKit::Bond::STEREOTRANS or stype == RDKit::Bond::STEREOE) {
```

Comment:
Looks like you've been switching between python and C++ too much, I've done that in the past as well. :)

Change the "or" to a "||"

Reply:
It kind of boggles my mind that gcc and clang now support this...

Reply:
Thanks!

It's technically valid C++ ! https://en.cppreference.com/w/cpp/keyword/or 

I wish I had a clang-format option to go one way or the other...